specification/monitor/resource-manager/Microsoft.Insights/stable/2018-09-01/metricBaselines_API.json

```diff
@@ -0,0 +1,381 @@
+        },
+        "interval": {
+          "type": "string",
+          "format": "duration",
+          "description": "The interval (window size) for which the metric data was returned in.  This may be adjusted in the future and returned back from what was originally requested.  This is not present if a metadata request was made."
+        },
+        "namespace": {
+          "type": "string",
+          "description": "The namespace of the metrics been queried."
+        },
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SingleMetricBaseline"
+          },
+          "description": "The baseline for each time series that was queried."
+        }
+      },
+      "required": [
+        "timespan",
+        "interval"
+      ],
+      "description": "A list of metric baselines."
+    },
+    "SingleMetricBaseline": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The metric baseline Id."
+        },
+        "type": {
+          "type": "string",
+          "description": "The resource type of the metric baseline resource."
+        },
+        "metricName": {
+          "type": "string",
+          "description": "The name of the metric for which the baselines were retrieved."
+        },
+        "baselines": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TimeSeriesBaseline"
+          },
+          "description": "The baseline for each time series that was queried."
+        }
+      },
+      "required": [
+        "id",
+        "type",
+        "metricName",
+        "baselines"
+      ],
+      "description": "The baseline results of a single metric."
+    },
+    "TimeSeriesBaseline": {
+      "properties": {
+        "aggregation": {
+          "type": "string",
+          "description": "The aggregation type of the metric."
+        },
+        "dimensions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MetricSingleDimension"
+          },
+          "description": "The dimensions of this time series."
+        },
+        "timestamps": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "format": "date-time",
+            "description": "The timestamp for the baseline value in ISO 8601 format."
+          },
+          "description": "The list of timestamps of the baselines."
+        },
+        "data": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SingleBaseline"
+          },
+          "description": "The baseline values for each sensitivity."
+        },
+        "metadataValues": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BaselineMetadata"
+          },
+          "description": "The baseline metadata values."
+        }
```

Comment:
Different from 2019-03-01, Please make sure you want to rename it.
https://github.com/Azure/azure-rest-api-specs/blob/521ed887e283fc4de186fa59c6b02cdbcd7f73fb/specification/monitor/resource-manager/Microsoft.Insights/stable/2019-03-01/metricBaselines_API.json#L223-L229

Reply:
checked. i'm sure.

Reply:
@asafst do you think we should rename 2019-03-01 from metadata to metadataValues as well? we're not flagged on this, but i think we could be.